src/GUI_draft/templates/create_project.html

```diff
@@ -23,6 +32,29 @@ <h4 class="cell"><a href="https://open-plan.readthedocs.io/en/latest/tool_interf
                 <input type="file" id="projParamsFile" class="show-for-sr">
             </div>
             <p class="cell">A list of inputs relevant for the project should be listed here</p>
+            <form class="cell small-9 grid-x">
+                {% for (param, param_details) in project_parameters.items() %}
+                    <div class="cell grid-x">
+                        <label class="cell small-6 input_field_labels">{{ param }}</label>
+                        {% for (param_info, value) in param_details.items() %}
+                            {% if param_info == 'docs_link' %}
```

Comment:
why do you loop here, you can assess the value directly with `param_details["docs_link"]`